kloppy/tests/prs/pr_358/test_pr_358.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime, timedelta, timezone
+from pathlib import Path
+
+import pytest
+from kloppy.domain import (
+    BodyPart,
+    BodyPartQualifier,
+    Point,
+    EventDataset,
+    SetPieceType,
+    SetPieceQualifier,
+    DatasetType,
+    DuelQualifier,
+    DuelType,
+    EventType,
+    GoalkeeperQualifier,
+    GoalkeeperActionType,
+    CardQualifier,
+    CardType,
+    Orientation,
+    PassResult,
+    FormationType,
+    Time,
+    PassType,
+    PassQualifier,
+)
+
+from kloppy import wyscout
+
+
+def test_ignore_unknown_player(base_dir):
+    dataset = wyscout.load(
+        event_data=base_dir / "prs" / "pr_358" / "wyscout_events_v3.json",
+        coordinates="wyscout",
+    )
```

Comment:
```
with pytest.warns(DeserializationWarning, match="..."):
    ...
```

Reply:
Haha, I didn't expect that this would work 😄 With "..." I actually meant that you should fill this in with the actual warning's text to make sure the correct one is printed.

Reply:
Apologies, I don't have experience with the warnings' module and wasn't paying enough attention 😄 

Should I then just fill in "Event" so it